.github/workflows/update-contribs.yml

```
@@ -0,0 +1,44 @@
+name: Update Contributors
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+jobs:
+    run-meta:
+        runs-on: ubuntu-latest
+        steps:
+        - name: Check out the code
+          uses: actions/checkout@v3
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: '3.10'
+        - name: Upgrade pip
+          run: |
+            # install pip=>20.1 to use "pip cache dir"
+            python3 -m pip install --upgrade pip
+            echo "$PWD"
+            git branch
+        - name: Install pyosmeta and run update contribs
+          env:
+            GITHUB_TOKEN: ${{ github.token }}
+          run: |
+            pip install pyosmeta@git+https://github.com/pyopensci/update-web-metadata
+            echo "$PWD"
+            python3 scripts/parse-contributors.py
```

I suggest running this with warnings enabled (unless there are too many out of your control to easily silence), so you'll get heads-up notice of deprecation, syntax problems and other issues before they turn into hard errors or cause undesired behavior (normally I'd add `-X dev` too for some additional checks, but didn't do so as it has a non-trivial performance impact and it seems this script is non-trivially expensive).

Also, drop the 3

```suggestion
            python -bb -W error scripts/parse-contributors.py
```